src/Core/src/Platform/Android/Resources/values/tags.xml

```diff
@@ -1,3 +1,2 @@
 <resources xmlns:android="http://schemas.android.com/apk/res/android">
-  <item name="automation_tag_id" type="id"/>
```

Comment:
Can you delete this whole file? Then it's one less `int` id, and file that `aapt2` would process?